### DISCUSSION TITLE
Midjourney style tuner alternative?

### DISCUSSION THREAD

**Discussion by aolko** [lllyasviel/Fooocus#861] [Q&A]
Is it possible for you to implement a midjourney style tuner alternative for fooocus?
https://twitter.com/nickfloats/status/1719936429000732836

**Comment by botzero-net** [comment]
Fooocus is already a better tuner IMO.  Your batch numbers can be as large or small as you like and you can easily add in extra styling, prompt adjustments, etc.

Once you have a style you like, find it in the history html file and copy its "style string" to the custom styles with its own name and checkbox on the styles tab.  I did this by copying an existing style json file in ./sdxl_styles and renaming the new file to sdxl_styles_customname.json and editing it to include the new style string with [prompt] (user specified) included.  Just follow the existing format.

Could maybe streamline the additions and/or include "style weighting" as a feature request, though.  I don't believe Fooocus has that--weighting one style differently than others when generating.

Midjourneyers are still fascinated by stylization and the idea of mixing styles because most of them have never used Fooocus.  :-)

**Reply by aolko** [reply]
While it's a useful advice, editing files by hand is a bit fiddly for most users, ui editor, or means to save the style in ui would be nice.

**Comment by lllyasviel** [comment]
style weighting may be added after 2.2
